toolchain/testing/file_test.cpp

```diff
@@ -88,7 +88,9 @@ class ToolchainFileTest : public FileTestBase {
  private:
   // Returns the toolchain subdirectory being tested.
   static auto GetComponent(llvm::StringRef test_name) -> llvm::StringRef {
-    CARBON_CHECK(test_name.consume_front("toolchain/"));
+    auto pos = test_name.find("toolchain/");
+    CARBON_CHECK(pos != llvm::StringRef::npos) << test_name;
+    test_name = test_name.drop_front(pos + sizeof("toolchain/") - 1);
```

Comment:
```suggestion
    test_name = test_name.drop_front(pos + strlen("toolchain/"));
```

Is this easier for you to read? I think the compiler-optimized result is equivalent for a string literal like this.